app/controllers/forms_controller.rb

```diff
@@ -7,6 +7,7 @@ def create
     form = Form.new({
       name: params[:name],
       submission_email: params[:submission_email],
+      org: current_user.organisation_slug,
```

Comment:
Just a sense check that we want to use org here rather than "organisation" - either works for me just wanting to make sure we're cool with it being a shortened version